static/js/admin.js

```diff
@@ -113,6 +117,14 @@ function populate_default_streams(streams_data) {
     loading.destroy_indicator($('#admin_page_default_streams_loading_indicator'));
 }
 
+function add_alias(alias_data) {
+    var del = '<td><button class="btn btn-danger btn-sm delete_alias" id="#id">Delete</button></td>';
+    var domain = '<td class="domain">'+alias_data[1]+'</td>';
+    var row = "<tr>"+domain+del.replace("#id", alias_data[0])+"</tr>";
```

Comment:
You should move this HTML construction into a handlebars template under static/templates; stylistically, we don't like to maintain raw HTML generated in JS since it's harder to grp and otherwise manage.